Docs/DafnyRef/DafnyRef.mdk

```diff
@@ -1005,21 +1008,33 @@ module Mod {
 Note that inside `m()`, we have to use `A` instead of `Helpers`, as we bound
 it to a different name. The name `Helpers` is not available inside `m()`,
 as only names that have been bound inside `Mod` are available. In order
-to use the members from another module, it either has to be declared
+to use the members from another module, that module either has to be declared
 there with `module` or imported with `import`.
 
 We don't have to give `Helpers` a new name, though, if we don't want
-to. We can write `import Helpers = Helpers` if we want to, and Dafny
-even provides the shorthand `import Helpers` for this behavior. You
+to. We can write `import Helpers = Helpers` to import the module under
+its own name; Dafny
+ provides the shorthand `import Helpers` for this behavior. You
 can't bind two modules with the same name at the same time, so
 sometimes you have to use the = version to ensure the names do not
 clash.
 
 The ``QualifiedModuleName`` in the ``ModuleImport_`` starts with a
 sibling module of the importing module, or with a submodule of the
-importing module. There is no wya to refer to the parent module, only
+importing module. There is no way to refer to the parent module, only
 sibling modules (and their submodules).
 
+Import statements may occur at the top-level as well. There they serve
+simply as a way to give a new name, perhaps
+a shorthand name, to a module. For example,
+
+```
+include "MyModule.dfy" // declares module MyModule
```

Comment:
Replace this `include` with the definition of module `MyModule` itself. The `include` feature is just a way to, essentially, add more files to the Dafny command line, and as such is unrelated to imports.